lib/Service/RoomService.php

```diff
@@ -362,6 +366,54 @@ public function setAvatar(Room $room, string $avatar): bool {
 		return true;
 	}
 
+	public function startRecording(Room $room, $status): bool {
+		$availableRecordingTypes = [Room::RECORDING_VIDEO, Room::RECORDING_AUDIO];
+		if (!in_array($status, $availableRecordingTypes)) {
+			throw new InvalidArgumentException('status');
+		}
+		if ($room->getCallRecording() !== 0) {
+			throw new InvalidArgumentException('room');
+		}
+		return $this->setCallRecording($room, $status);
+	}
+
+	public function stopRecording(Room $room): bool {
+		if ($room->getCallRecording() === 0) {
+			throw new InvalidArgumentException('room');
+		}
+		return $this->setCallRecording($room);
+	}
+
+	/**
+	 * @param Room $room
+	 * @param integer $status 0 none|1 video|2 audio
+	 */
```

Comment:
```suggestion
```
Let's skip this until 8.1 is minimum with proper ENUM support?

Reply:
I want this!
When we will migrate to PHP 8.1?

Reply:
End of next year 🙈